src/algebra/add_torsor.lean

```diff
@@ -335,7 +335,7 @@ def const_vadd_hom : multiplicative G →* equiv.perm P :=
 
 variable {P}
 
-open function
+open _root_.function
```

Comment:
Is the `_root_` necessary, or is it vestigial?

Reply:
It is necessary, for reasons I don't understand.

Reply:
Oh, it's because there's an `equiv.function` namespace. I didn't know that `_root_` worked in `open` commands.

I suppose this is fine, but I checked that you can put `open function` near the beginning of the module. If you don't think that needs to be done, then the PR looks good to me.

Reply:
I would prefer to keep it like this.